docs/detections/about-rules.asciidoc

```diff
@@ -42,6 +42,10 @@ TIP: You can also use value lists as the indicator match index. See <<indicator-
 
 * <<create-new-terms-rule, *New terms*>>: Generates an alert for each new term detected in source documents within a specified time range. You can also detect a combination of up to three new terms (for example, a `host.ip` and `host.id` that have never been observed together before).
 
+* <<create-esql-rule, *ES|QL*>>: Uses an {ref}/esql.html[Elasticsearch Query Language (ES|QL)] query to search your events and aggregate source event data. 
```

Comment:
Yeah, good point! The current description isn't all that informative if you're wondering how ES|QL rules create alerts. Initially, I had considered writing a general statement like "creates and alert when the query is met", but I decided against it because I didn't think it was descriptive enough. I go into more detail about how alerts are generated in the actual steps though, so writing high-level explanation is probably fine here.

```suggestion
* <<create-esql-rule, *ES|QL*>>: Searches the defined indices and creates an alert when results match an {ref}/esql.html[Elasticsearch Query Language (ES|QL)] query.
```

Reply:
Sounds good, thanks for the context!